### DISCUSSION TITLE
Labels triggering math commands

### DISCUSSION THREAD

**Discussion by liris8** [sleepymalc/VSCode-LaTeX-Inkscape#30] [Q&A] [answered]
Hi, I implemented your more generic version of the math context:
function math(context) {
    return  context.scopes.findLastIndex(s => s.startsWith("meta.math")) > context.scopes.findLastIndex(s => s.startsWith("comment") || s.startsWith("meta.text.normal.tex"));
}
the \text{} works right in the math context but in the labels of equations the math snippets keep triggering. Is there a way of fixing this?

**Comment by sleepymalc** [comment, marked as answer]
I'm unaware of a solution to this since this is based on the scope, and I don't think the label has a corresponding scope.

**Reply by liris8** [reply]
do you deal with this in a certain way?

Thank you very much for all your quick answers and all the help 🙏🏽

**Reply by sleepymalc** [reply]
Sadly no. The workaround I'm using is just to write things outside the math environment and cut-paste them to the correct place.
